app/services/api/v3/ecf/participants_query.rb

```diff
@@ -62,7 +62,9 @@ def from_participant_id
         end
 
         def latest_induction_records_join
-          super
+          InductionRecord
+            .select("induction_records.id AS latest_id")
+            .joins("JOIN (#{super.to_sql}) AS latest_induction_records_join ON latest_induction_records_join.latest_id = induction_records.id")
             .joins(:schedule)
```

Comment:
Could this not be included in `participants_for_pagination` instead and we can leave `latest_induction_records_join` as it is without cohort, as we're doing the latest induction record join twice?